types/react-stripe-elements/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-stripe-elements 1.1
+// Type definitions for react-stripe-elements 1.2
```

Comment:
This version isn't supposed to be incremented unless it's to reflect version 1.2 of the library.

Also, this library is at version 3.0 so can you clarify if the major version should be changed?

Reply:
Ah I see. It looks like it should be 3.0, as the signature of `createPaymentMethod` I added to the types is consistent with the 3.0 version. Most of the rest remains compatible, but there is one breaking change: the `PostalCodeElement` class was removed in 3.0. Should I also remove that one when I bump the version?